src/style.css

```diff
@@ -0,0 +1,114 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+body {
+  font-family: Avenir, sans-serif;
```

Comment:
add fallback font

Reply:
What does it mean ?

Reply:
`font-family: Avenir, Arial, sans-serif;`